Sources/WordPressData/Swift/Post+CoreDataProperties.swift

```diff
@@ -27,3 +27,20 @@ public extension Post {
     @NSManaged func addCategories(_ values: Set<PostCategory>)
     @NSManaged func removeCategories(_ values: Set<PostCategory>)
 }
+
+extension Post {
+    public var allowComments: Bool {
+        get { commentsStatus != DiscussionState.closed.rawValue }
+        set { commentsStatus = (newValue ? DiscussionState.open : .closed).rawValue }
+    }
+
+    public var allowPings: Bool {
+        get { pingsStatus != DiscussionState.closed.rawValue }
+        set { pingsStatus = (newValue ? DiscussionState.open : .closed).rawValue }
+    }
+
+    private enum DiscussionState: String, CaseIterable {
+        case open = "open"
+        case closed = "closed"
```

Comment:
Make the enum public, and use it to replace the hard-coded "open" & "closed" strings in other places?